core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```diff
@@ -48,7 +48,7 @@ <h1 class="stat-value">{{ subscriptionsList && subscriptionsList.length }}</h1>
         </p>
         <a class="btn oppia-learner-dashboard-intro-button oppia-transition-200 protractor-mobile-test-learner-dashboard-intro-button"
            [innerHTML]="'I18N_ACTION_BROWSE_LESSONS' | translate"
-           [href]="community-library"
+           href="/community-library"
```

Comment:
Can we use this instead `[href]="'community-library'"` ?

Reply:
Both approaches are correct? Convince me why yours is better.

Reply:
my approach does not require angular to do the compilation of the attribute.
I have checked everywhere in the codebase,  `/` is prefixed to `community-library` everywhere. So, I am sticking with the convention.

Reply:
After the completion of migration, we shouldn't be using href.
(we should use [href])
As it will reload the entire page which is not ideal.

/cc @kevintab95 @nithusha21 @srijanreddy98

Reply:
I think routerLink does that.

Reply:
Both `[href]="'abc'"` and `href='abc'` will do the same thing with current settings we have. We typically use the former when we have to bind to a variable from the component. Since it's not the case here, it would be better to use `href='/abc'`. Once we shift to angular-router, we change the links.

Does this answer your query @Radesh-kumar?

Reply:
Got it thanks!